### PR TITLE
Special cov preview publish

### PR DESCRIPTION
**Pull request by benghaziboy**
@mparent61 @MichaelButkovic

### CODE REVIEW

File: tests/special_coverage/test_spec_cov_es.py [hunk 1]
```diff
@@ -85,9 +83,7 @@ def test_save_percolator(self):
         )
         # Shutting up publishing
 
-        sc_query = sc.get_content().to_dict()["query"]
-        del sc_query["filtered"]["filter"]["bool"]["must"][1]
-        del response["_source"]["query"]["filtered"]["filter"]["bool"]["must"][1]
+        sc_query = sc.get_content(published=False).to_dict()["query"]
```
Comment (mparent61) [inline tests/special_coverage/test_spec_cov_es.py]: Do you have any logic to test get_content(published=True)?

Comment (benghaziboy) [inline tests/special_coverage/test_spec_cov_es.py]: Already working on it mama.

**Comment by mparent61** [comment]
What is the use-case for this behavior?

**Comment by mparent61** [comment]
Whoops
:+1:

**Comment by MichaelButkovic** [comment]
LGTM

**Comment by benghaziboy** [comment]
I'm glad we have documented @mparent61 's incompetence

**Comment by mparent61** [comment]
Nothing to see here

**Pull request merged**